src/Util.hs

```diff
@@ -62,10 +62,7 @@ customBuild rs opts target@Target {..} = do
                     top   <- topDirectory
                     cmd [Cwd output] [path] "x" (top -/- input)
 
-            Configure dir -> do
-                -- Inject /bin/bash into `libtool`, instead of /bin/sh
-                let env = AddEnv "CONFIG_SHELL" "/bin/bash"
```

Comment:
Removing this breaks the Windows build (the CI failed).

I agree that this bit is mysterious and needs to be better documented and perhaps slightly refactored. Could you inject `bashPath` here and add a TODO for me?